pkg/odo/util/cmdutils.go

```diff
@@ -51,17 +54,37 @@ Use "{{.CommandPath}} [command] --help" for more information about a command.{{e
 `
 
 // PrintComponentInfo prints Component Information like path, URL & storage
-func PrintComponentInfo(currentComponentName string, componentType string, path string, componentURL string, appStore []storage.StorageInfo) {
+func PrintComponentInfo(currentComponentName string, componentDesc component.ComponentDesc) {
+	fmt.Printf("Component Name: %v\nType: %v\n", currentComponentName, componentDesc.ComponentImageType)
 	// Source
-	if path != "" {
-		fmt.Println("Component", currentComponentName, "of type", componentType, "with source in", path)
+	if componentDesc.Path != "" {
+		fmt.Printf("Source: %v\n", componentDesc.Path)
 	}
-	// URL
-	if componentURL != "" {
-		fmt.Println("Externally exposed via", componentURL)
+
+	// Env
+	if componentDesc.Env != nil {
+		fmt.Println("\nEnvironment variables:")
+		for _, env := range componentDesc.Env {
+			if !strings.Contains(env.Name, "ODO") {
```

Comment:
Since `componentDesc.Env` is already filtering env variables with `ODO` in their name (if I read https://github.com/redhat-developer/odo/pull/1046/files#diff-692effa1b0b7a9ed8c1f7bb49a977b9aR690 correctly), do we really need this check here?

Reply:
Also, perhaps replace `Env` by `FilteredEnv` in `ComponentDesc` and add `Env` to have *all* env variables without any filtering?

Reply:
done